tc_driver/dycore.jl

```diff
@@ -129,19 +129,8 @@ function compute_explicit_gm_tendencies!(
     aux_tc = TC.center_aux_turbconv(state)
 
     # Apply precipitation tendencies
-    @. tendencies_gm.ρq_tot +=
-        ρ_c * (
-            aux_bulk.qt_tendency_precip_formation +
-            aux_en.qt_tendency_precip_formation +
-            aux_tc.qt_tendency_precip_sinks
-        )
```

Comment:
I'm realizing now, it doesn't look like this is equivalent to where the tendencies are now being added (in `src/tendencies/precipitation.jl`):

```julia
        @. Yₜ.c.ρe_tot[colidx] +=
            ᶜS_ρq_tot[colidx] * (
                ᶜλ[colidx] *
                TD.internal_energy_liquid(thermo_params, ᶜts[colidx]) +
                (1 - ᶜλ[colidx]) *
                TD.internal_energy_ice(thermo_params, ᶜts[colidx]) +
                ᶜΦ[colidx]
            )
```
because, in this PR, it's getting multiplied by the `(
                ᶜλ[colidx] *
                TD.internal_energy_liquid(thermo_params, ᶜts[colidx]) +
                (1 - ᶜλ[colidx]) *
                TD.internal_energy_ice(thermo_params, ᶜts[colidx]) +
                ᶜΦ[colidx]
            )` factor

Reply:
I think it's the same: https://github.com/CliMA/ClimaAtmos.jl/blob/fcc815d004f8f32bf04440e89251077b5a59fdff/src/TurbulenceConvection/microphysics_coupling.jl#L108